test/conformance/device/urDeviceGetInfo.cpp

```diff
@@ -101,6 +104,10 @@ static std::unordered_map<ur_device_info_t, size_t> device_info_size_map = {
     {UR_DEVICE_INFO_BFLOAT16, sizeof(bool)},
     {UR_DEVICE_INFO_MAX_COMPUTE_QUEUE_INDICES, sizeof(uint32_t)},
     {UR_DEVICE_INFO_KERNEL_SET_SPECIALIZATION_CONSTANTS, sizeof(bool)},
+    {UR_DEVICE_INFO_MEMORY_BUS_WIDTH, sizeof(bool)},
+    {UR_DEVICE_INFO_MAX_WORK_GROUPS_3D, sizeof(uint32_t)},
+    {UR_DEVICE_INFO_ASYNC_BARRIER, sizeof(bool)},
+    {UR_DEVICE_INFO_MEM_CHANNEL_SUPPORT, sizeof(bool)},
```

Comment:
I beleive all instances of `bool` here should actually be `ur_bool_t`, the are the same size but since we are testing the spec we should use exactly the same type. Might as well fix them all while touching this file.

Reply:
https://github.com/oneapi-src/unified-runtime/pull/460 was did a similar cleanup.

Reply:
Just copied what was already in the file but that makes sense! Changed them over now, squashed my commits down to make it a little cleaner.